packages/router/router.ts

```diff
@@ -1805,25 +1888,15 @@ export function unstable_createStaticHandler(
 
     // Pick off the right state value to return
     let routeData = [result.actionData, result.loaderData].find((v) => v);
-    let value = Object.values(routeData || {})[0];
-
-    if (isRouteErrorResponse(value)) {
-      return new Response(value.data, {
-        status: value.status,
-        statusText: value.statusText,
-      });
-    }
```

Comment:
Not a possible path either since all `isRouteRequest`'s throw inside `callLoaderOrAction`